### PR TITLE
[SMALLFIX]Sort the property in letter order

### PR DESCRIPTION
**Pull request by maobaolong**

### CODE REVIEW

**Comment by maobaolong** [comment]
it was not caused by me.

**Comment by yupeng9** [comment]
jenkins, test this please

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 1]
```diff
@@ -297,6 +300,10 @@
   USER_NETWORK_NETTY_READER_BUFFER_SIZE_PACKETS(Name.USER_NETWORK_NETTY_READER_BUFFER_SIZE_PACKETS,
       16),
   USER_NETWORK_NETTY_READER_CANCEL_ENABLED(Name.USER_NETWORK_NETTY_READER_CANCEL_ENABLED, true),
+  USER_PACKET_STREAMING_ENABLED(Name.USER_PACKET_STREAMING_ENABLED, false),
+  USER_RPC_RETRY_BASE_SLEEP_MS(Name.USER_RPC_RETRY_BASE_SLEEP_MS, 50),
+  USER_RPC_RETRY_MAX_SLEEP_MS(Name.USER_RPC_RETRY_MAX_SLEEP_MS, 5000),
```
Comment (yupeng9) [inline core/common/src/main/java/alluxio/PropertyKey.java]: swap with the line below?

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 2]
```diff
@@ -763,6 +770,14 @@
         "alluxio.user.network.netty.reader.buffer.size.packets";
     public static final String USER_NETWORK_NETTY_READER_CANCEL_ENABLED =
         "alluxio.user.network.netty.reader.cancel.enabled";
+    public static final String USER_PACKET_STREAMING_ENABLED =
+        "alluxio.user.packet.streaming.enabled";
+    public static final String USER_RPC_RETRY_BASE_SLEEP_MS =
+        "alluxio.user.rpc.retry.base.sleep.ms";
+    public static final String USER_RPC_RETRY_MAX_SLEEP_MS =
```
Comment (yupeng9) [inline core/common/src/main/java/alluxio/PropertyKey.java]: ditto

**Comment by maobaolong** [comment]
@yupeng9 PTAL.

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 2 above)
Comment (maobaolong) [inline core/common/src/main/java/alluxio/PropertyKey.java]: done.

File: core/common/src/main/java/alluxio/PropertyKey.java (hunk 1 above)
Comment (maobaolong) [inline core/common/src/main/java/alluxio/PropertyKey.java]: done.

**Pull request merged**